tests_integration/spcs/test_image_repository.py

```diff
@@ -9,9 +9,9 @@
 INTEGRATION_REPOSITORY = "snowcli_repository"
 
 
-@pytest.mark.integration
+@pytest.mark.integration_disable
```

Comment:
Why not use `itegration` and `pytests.mark.skip` instead of custom marker?

Reply:
fair point, these changes were temporary but I will note for the future to use pytest.mark.skip instead